community-operators/tidb-operator/tidb-operator.package.yaml

```diff
@@ -2,4 +2,4 @@ packageName: tidb-operator
 defaultChannel: stable
 channels:
   - name: stable
-    currentCSV: tidb-operator.v1.1.2
+    currentCSV: tidb-operator.v1.1.5
```

Comment:
You are adding new version of the operator, while v1.1.2 is the previous version, for the package file you need keep 1.1.2 and add new entry in the channels list, this might resolve the opm index build error in CI.

For example:
https://github.com/operator-framework/community-operators/blob/master/community-operators/3scale-community-operator/3scale-community-operator.package.yaml

Reply:
Thanks, it appears the new CSV for `v1.1.5` need to have `replaces: tidb-operator.v1.1.2` included. I failed to do so with https://operatorhub.io/packages as it deletes all content of the new CSV file if I use "Edit Upgrade Graph" to set replace version, and I had to manually edit the CSV file to add that line.